_events/sr2022/competition.md

```diff
@@ -31,9 +31,16 @@ We have a limited amount of free parking on site for competitors.
 
 ![parking information]({{ '/resources/sr2022/parking-map.png' | prepend: site.baseurl }})
 
+## Media Consent
+
+Before gaining access to the venue, everyone will need to sign a [media consent][mcf-link]). Please put "Student Robotics 2019" as the event name. Those under the age of 16 will need to get a parent or guardian to sign on their behalf.
```

Comment:
```suggestion
Before gaining access to the venue, everyone will need to sign a [media consent][mcf-link]). Please put "Student Robotics 2022" as the event name. Those under the age of 16 will need to get a parent or guardian to sign on their behalf.
```

Reply:
Please check where they will be putting the event...

Reply:
```suggestion
Before gaining access to the venue, everyone will need to sign a [media consent][mcf-link]). Those under the age of 16 will need to get a parent or guardian to sign on their behalf.
```

Reply:
```suggestion
Before gaining access to the venue, everyone will need to sign a [media consent form][mcf-link]. Those under the age of 16 will need to get a parent or guardian to sign on their behalf.
```